packages/react-flicking/src/react-flicking/Flicking.tsx

```diff
@@ -133,6 +147,7 @@ class Flicking extends React.Component<Partial<FlickingProps & FlickingOptions>>
     this._checkPlugins();
     renderEmitter.trigger("render");
     flicking.camera.updateOffset();
+    this._renderingState = RenderingState.UPDATED;
```

Comment:
If it is used like this, does renderingState seem to have no meaning?

Reply:
It's meant to prevent updates in `beforeRender` when there're two render calls without calling  `componentDidUpdate`.

Reply:
Have you checked codesandbox with the fixed code?

Reply:
> ## Issue
> #736
> 
> ## Details
> This fixes #736
> 
> Tested this doesn't produce any error furthermore on both this two test cases:
> 
> * https://github.com/neizzz/flicking-bug-reproduction
> * https://codesandbox.io/s/flicking-change-data-bug-voblvn

Yes.

But, the commit you suggested https://github.com/naver/egjs-flicking/pull/739/commits/852d63f76d65e32f723212dc8e0eb59bad902f8e actually broke that case.
I found the reason & I'll revert that with more explanation of it.